### PR TITLE
change the counter to number of insured people with signed insurances

### PR DESCRIPTION
**Pull request by meletios**

### CODE REVIEW

File: src/main/java/com/hedvig/productPricing/service/query/ProductRepository.java [hunk 1]
```diff
@@ -66,7 +66,7 @@
     @Param("firstDayOfNextMonth") LocalDateTime firstDayOfNextMonth, @Param("memberId") String memberId
   );
 
-  @Query("select count(*) from ProductEntity p where p.state = 'SIGNED' and (p.activeTo is null or p.activeTo > current_date)")
+  @Query("select count(p.personsInHouseHold) from ProductEntity p where p.state = 'SIGNED' and (p.activeTo is null or p.activeTo > current_date)")
```
Comment (vonElfvin) [inline src/main/java/com/hedvig/productPricing/service/query/ProductRepository.java]: About this query, isn't it missing "where activeFrom IS NOT NULL"

Comment (vonElfvin) [reply src/main/java/com/hedvig/productPricing/service/query/ProductRepository.java]: Also 
```suggestion
  @Query("select sum(p.personsInHouseHold) from ProductEntity p where p.state = 'SIGNED' and (p.activeTo is null or p.activeTo > current_date)")
```

Comment (vonElfvin) [inline src/main/java/com/hedvig/productPricing/service/query/ProductRepository.java]: I.e.

```suggestion
  @Query("select sum(p.personsInHouseHold) from ProductEntity p where p.state = 'SIGNED' and (p.activeTo is null or p.activeTo > current_date) and p.activeFrom IS NOT NULL")
```

Comment (akashpatl) [reply src/main/java/com/hedvig/productPricing/service/query/ProductRepository.java]: nice catch @vonElfvin

**Comment by meletios** [comment]
I am closing this and deleting the branch until we clear the requirements for the counter. 👋

**Pull request closed without merging**